spec/mutations/users/register_user_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+module Mutations
+  module Users
+    RSpec.describe RegisterUser, type: :request do
+      describe '.resolve' do
+        it 'creates a new user' do
+          @input = {
+                    image: "www.image.com",
+                    firstName: "Test",
+                    lastName: "User",
+                    state: "CO",
+                    city: "Denver",
+                    street: "1234 Road Street",
+                    zip: "1234",
+                    authProvider: {
+                      credentials: {
+                        username: "AnothaOne",
+                        password: "[omitted]"
```

Comment:
Since you were able to use bcrypt to handle the auth password, would the password string just be `"[omitted]"` for tests that use authProvider credentials? Is this how we're mocking the password? If so, I can add to the update user mutation testing because I initially had the password field commented out.

Reply:
It can actually be anything right now. That's just what I stole from a tutorial and figured it worked. I don't think it matters regarding testing?